src/utils/backend/vaccine.ts

```diff
@@ -0,0 +1,86 @@
+import { BackendDataReturn, DatabaseClient } from "@utils/types/common";
+import { VaccineRecord } from "@utils/types/vaccine";
+
+export async function getVaccineRecordbyPersonId(
+  supabase: DatabaseClient,
+  personId: number
+): Promise<VaccineRecord[]> {
+  const { data, error } = await supabase
+    .from("vaccine_records")
+    .select("*")
+    .eq("person", personId)
+    .order("vaccination_date", { ascending: true });
+
+  if (error) {
+    // throw error;
+    console.error(error);
```

Comment:
This is not what I meant. We’re using `BackendReturn` and `BackendDataReturn` for most backend functions right now. This ensures all backend functions have predictable returns.

None of the functions here use it correctly. I’ll let it be your call to fix this in this PR or not.